upup/pkg/fi/cloudup/populate_instancegroup_spec.go

```diff
@@ -49,8 +50,10 @@ const (
 	defaultMasterMachineTypeDO      = "s-2vcpu-4gb"
 	defaultMasterMachineTypeAzure   = "Standard_B2s"
 	defaultMasterMachineTypeHetzner = "cx21"
+	defaultMasterMachineTypeSCW     = "DEV1-M"
 
-	defaultDONodeImage = "ubuntu-20-04-x64"
+	defaultDONodeImage  = "ubuntu-20-04-x64"
+	defaultSCWNodeImage = "debian_buster"
```

Comment:
Why not the latest ubuntu? or the latest debian?

Reply:
We prefer Ubuntu 20.04 for now:
https://github.com/kubernetes/kops/issues/14140